proto/api/v1/api_token_service.proto

```diff
@@ -2,18 +2,21 @@ syntax = "proto3";
 
 package v1;
 
+import weak "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
 import "api/v1/common.proto";
 import "api/v1/empty.proto";
-import weak "google/api/annotations.proto";
```

Comment:
Style checks seem to complain about the order of the imports and the indentation in the altered message structure. Can you please fix that ?